src/box2d-netstandard.sln

```diff
@@ -1,40 +1,49 @@
-﻿
```

Comment:
Sorry, why `box2d-netstandard.sln` was changed? It works by keeping it as is. I'd vote to leave it unchanged.

Projects in `./examples` and `./src` are separated projects, and only `./examples` projects can reference `./src` projects. If you'd like to have one project solution file, then please just create `box2d.sln` in the root directory.